### PR TITLE
Add Katello 3.8 configuration

### PR DESCRIPTION
**Pull request by ehelms**
@jlsherrill Mind vetting?

### CODE REVIEW

File: configs/katello/3.8.yaml [hunk 1]
```diff
@@ -0,0 +1,178 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.8.0:
```
Comment (ekohl) [inline configs/katello/3.8.yaml]: Should this be 3.7.0?

File: configs/katello/3.8.yaml [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    :redmine_release_id: 338
+:releases:
+  :3.8.0:
+    :redmine_release_id: 356
```
Comment (jlsherrill) [inline configs/katello/3.8.yaml]: we should go ahead and add 3.7.1 (and maybe 3.7.0) to this

File: configs/katello/3.8.yaml [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    build_target: katello-3.8-rhel7-build
+    build_package_group_source_tag: katello-nightly-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.8-rhel7-build:
+        katello-3.8-rhel7-override: 0
+        foreman-plugins-1.18-rhel7-override: 3
+        foreman-1.18-nonscl-rhel7: 4
+        foreman-1.18-rhel7: 10
```
Comment (jlsherrill) [inline configs/katello/3.8.yaml]: these foreman tags should be 1.19 right?

File: configs/katello/3.8.yaml [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+      katello-3.8-client-rhel7:
+        katello-3.8-rhel7: 0
+        katello-3.8-thirdparty-pulp-rhel7: 2
+    external_repos:
+      - epel-7
+      - centos-sclo-rh-rhel-7
+      - centos-sclo-sclo-rhel-7
+      - centos-7-server
+      - rhel-7.0-server
+      - foreman-rails-1.18-rhel-7
```
Comment (jlsherrill) [inline configs/katello/3.8.yaml]: 1.19 ?

**Comment by ehelms** [comment]
@jlsherrill updated

**Pull request merged**